### PR TITLE
Change the format of the ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by sayanchowdhury**
Change the ISSUE_TEMPLATE to group similar release items and group them, so that the ownership is group based rather than line-item based.

Opening the PR to get feedback from team and change how it fits best for the team during releases.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (sayanchowdhury) [inline .github/ISSUE_TEMPLATE/new_release.md:79]: @tormath1 Do you think this format is good?

Comment (sayanchowdhury) [reply .github/ISSUE_TEMPLATE/new_release.md:79]: This is in case there are other providers, but I'm fine with your suggestion too.

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (sayanchowdhury) [inline .github/ISSUE_TEMPLATE/new_release.md:79]:
```suggestion
- C. Cluster API for new Stable
  - [ ] Azure: Manually run `./azure-sig.sh` to create New Shared Community Gallery Images
  - [ ] OpenStack: [documentation](https://gist.github.com/tormath1/acbae5c6cd12420bb8ea137e25655c99)
```

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (pothos) [inline .github/ISSUE_TEMPLATE/new_release.md:20]: Maybe the https://hackmd.io/team/flatcar?nav=overview link is useful for those that have to create one and don't remember how

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (pothos) [inline .github/ISSUE_TEMPLATE/new_release.md:41]:
```suggestion
- [ ] C. Release notes / announcements added to Go/No Go doc from `container/image_changes` job by selecting `Timestamps: None`.
```

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (pothos) [inline .github/ISSUE_TEMPLATE/new_release.md:50]:
```suggestion
- [ ] A. Created the Release Notes on the `scripts` repo for the respective tags on GitHub.
```

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (pothos) [inline .github/ISSUE_TEMPLATE/new_release.md:62]: The merging could be moved as last point where the website PR gets created - the owner should be the same. It's also ok to have it published then because Origin already provides the newer images then for `flatcar-install`. Even if not, website publishing upfront means that people can become aware of the incoming update earlier.

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (sayanchowdhury) [reply .github/ISSUE_TEMPLATE/new_release.md:20]: Ah! yes. This link is in local, maybe I forgot to push. I will push the change.

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (sayanchowdhury) [reply .github/ISSUE_TEMPLATE/new_release.md:62]: Yes, I went ahead and divided into two task, as I usually wait for the Release package publishing before merging the website PR but your argument is also correct. I will change to your suggestion.

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (sayanchowdhury) [reply .github/ISSUE_TEMPLATE/new_release.md]: You are right! I've merged them as per your suggestions.

File: .github/ISSUE_TEMPLATE/new_release.md
Comment (dongsupark) [inline .github/ISSUE_TEMPLATE/new_release.md:54]: You would probably mean "Website PR created with `./update-flatcar-versions.sh` and requested review."?

**Review by dongsupark — APPROVED** [review]

**Pull request merged**
